tests/openwisp2/settings.py

```diff
@@ -164,9 +164,13 @@
 OPENWISP_ORGANIZATION_OWNER_ADMIN = True
 OPENWISP_USERS_AUTH_API = True
 
-CELERY_TASK_ALWAYS_EAGER = True
```

Comment:
It's not great to do this here, we run all tests in eager mode across the repositories we maintain, why did you have to do this? Are you 100% is required? This is going to add overhead.

Reply:
Yes, in tests we are still using [CELERY_TASK_ALWAYS_EAGER = True](https://github.com/openwisp/openwisp-network-topology/blob/0e23f1cccb284395e5541ac5b7f97a6d68762da7/tests/openwisp2/settings.py#L167-L172). I have used similar settings as those in other modules. Please see below:
 
https://github.com/openwisp/openwisp-controller/blob/a0c818a365c4dafcb1692653804d7d653e540540/tests/openwisp2/settings.py#L171-L176

https://github.com/openwisp/openwisp-monitoring/blob/25e00a23be37761622b1551419c63634b6c65f00/tests/openwisp2/settings.py#L168-L173

Reply:
ok I read this too quckly, thanks!